OneCommunityGlobal/HighestGoodNetworkApp
# No badges will show to be assigned

# Description

Fixes 11 ( BADGES BUGS - PRIORITY - MEDIUM )
 ‘Welcome, Username’ > View Profile > Featured Badges > Assign Badges
If a user hasn’t visited: Other Links > Badge Management yet, no badges will show to be assigned.


## Mainly changes explained:
1 - Added a function called `allBadges( )` that load all badges to be assigned in the component AssignBadgePopup.jsx
2 - Called this function inside `useEffect( )` to load all badges when the modal opens.
…

## How to test:
check into current branch 
do `npm install` and ` npm run start:local` to run this PR locally
log as admin user then:
‘Welcome, Username’ → View Profile →Featured Badges → Assign Badges
See the bagdes will load.
![Captura de Tela (123)](https://user-images.githubusercontent.com/39320057/211076147-826e3f7d-14b3-45ed-998d-60377f3f914f.png)


https://user-images.githubusercontent.com/39320057/211075831-665f3e5b-2c43-46a8-94fc-970317d16819.mp4
